modules/navier_stokes/test/tests/finite_volume/ins/mms/skew-correction/tests

```diff
@@ -1,11 +1,15 @@
 [Tests]
   issues = '#16239'
   design = 'navier_stokes/insfv.md'
-  [skewness-error]
-    type = RunException
-    input = 2d-rc.i
+  [vortex-skewness-corrected]
+    type = PythonUnitTest
+    input = test.py
+    test_case = TestVortexSkewCorrected
+    method = '!dbg'
+    min_parallel = 2
+    requirement = 'The system shall be able to solve the incompressible Navier-Stokes equations on triangular meshes, using a Rhie-Chow scheme and skewness-correction, Dirichlet boundary conditions for the velocity, and demonstrate second order convergence for the velocity and first order convergence for pressure.'
     ad_indexing_type = 'global'
-    requirement = "The system shall trow an error if the cell gradient is computed with skewness-correction in the INSFVVelocity variable."
-    expect_err = "INSFVelocityVariable does not support skewness-correction at this point!"
+    required_python_packages = 'pandas matplotlib'
+    heavy = true
```

Comment:
Does this actually have to be marked `heavy`? How long does this test take?

Reply:
The whole mms case takes about 50 s on my local machine.

Reply:
What if you use like 8 procs?

Reply:
With 8 procs it is 11 s.

Reply:
Ok I think I'm fine with 2 and running heavy or going with 8 and running non-heavy. Up to you

Reply:
I'll keep it heavy with 2.